docs/libcurl/curl_easy_setopt.3

```diff
@@ -684,6 +684,8 @@ Share object to use. See \fICURLOPT_SHARE(3)\fP
 Mode for creating new remote files. See \fICURLOPT_NEW_FILE_PERMS(3)\fP
 .IP CURLOPT_NEW_DIRECTORY_PERMS
 Mode for creating new remote directories. See \fICURLOPT_NEW_DIRECTORY_PERMS(3)\fP
+.IP CURLOPT_MAY_LEAK
```

Comment:
I think "may leak" is a weird name of the option since that's just an effect of what you're asking for. You're actually asking for libcurl to *not* wait around (for threads) when closing down a transfer. Maybe "fast exit", "quick exit" or "fast shutdown" or something is a better name?

Reply:
DONE: "quick exit"

Reply:
g